modules/apps/asset/asset-list-service/src/test/java/com/liferay/asset/list/internal/exportimport/content/processor/AssetListEntryExportImportContentProcessorTest.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.asset.list.internal.exportimport.content.processor;
+
+import com.liferay.exportimport.kernel.lar.PortletDataContext;
+import com.liferay.portal.kernel.service.GroupLocalService;
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.xml.Element;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+/**
+ * @author Gustavo Lima
+ */
+public class AssetListEntryExportImportContentProcessorTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Before
+	public void setUp() {
+		_assetListEntryExportImportContentProcessor =
+			new AssetListEntryExportImportContentProcessor();
+	}
+
+	@Test
+	public void testAddGroupMappingsElementWithInexistentGroup() {
```

Comment:
I don't really like this test, it seems too narrow to check for real problems. Could you look into making this an integration (or playwright) test if necessary to more realistically test the use case.

Reply:
i think the unit test is probably fine, lets just add another assert: addElement() is called once when when group is not null